core/variable_binary_arithmetic.cpp

```diff
@@ -64,18 +64,18 @@ template <class T> class hasReciprocal {
       !std::is_same_v<void, decltype(detect(std::declval<T>()))>;
 };
```

Comment:
You can also remove this, and the addition of `reciprocal` for `ValueAndVariance`.